source/reference/minio-mc/mc-support-logs-show.rst

```diff
@@ -0,0 +1,100 @@
+========================
+``mc support logs show``
+========================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+.. mc:: mc support logs show
+
+Description
+-----------
+
+Use the :mc-cmd:`mc support logs show` command to display MinIO server logs.
+   
+.. include:: /includes/common-mc-support.rst
+   :start-after: start-minio-only
+   :end-before: end-minio-only
+
+Examples
+--------
+
+Show Logs for the Alias ``minio1``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following command shows logs for the alias ``minio1``.
+
+.. code-block:: shell
+   :class: copyable
+
+   mc support logs show minio1
+
+Show Last Five Log Entries for a Specific Node
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following command shows the last five log entries on the node ``node1`` for a MinIO server with the alias ``minio1``.
+
+.. code-block:: shell
+   :class: copyable
+
+   mc support logs show --last 5 minio1 node1
+
+Show Application Log Entries
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following command shows the application errors in logs for a MinIO server with the alias ``minio1``.
```

Comment:
only errors or any general client op?

Reply:
It is expected that MinIO logs contain only errors, but yes - better to just say `logs` instead of `errors in logs`.

Reply:
Removed "errors in"